src/popover/popover.ts

```diff
@@ -43,7 +43,7 @@ let nextId = 0;
   },
   template: `
     <div class="arrow"></div>
-    <h3 class="popover-header">
+    <h3 class="popover-header" *ngIf="title">
```

Comment:
Could you please change it to `title != null` (which will catch both `null` and `undefined`) to be explicit and avoid all discussion around passing `0, ""` and other falsy values?

Reply:
Good point; done.